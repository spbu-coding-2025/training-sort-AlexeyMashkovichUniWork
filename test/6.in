511151302 -119720469 586729240 -733348278 79384943 -303129302 -245403077 944157591 58796479 591589240 -692148342 1052208971 -408512545 960547330 -433427300 283646186 469757932 -773010325 280523379 -896410198 249810896 533337182 422712389 266163929 125407446 408713394 -983560054 -436421619 1018805020 975481150 -136204352 456214497 -217981144 -623216937 796608042 935145623 147395584 -522536860 805561389 -867549762 -1004689445 -960328778 -889082616 -339460166 -1073523273 -248768092 1017927843 469976483 51963406 224709397 647308108 -771967523 -315695246 -3721327 567938229 883454023 -668749758 658119998 -626709420 -723686563 559859324 310828051 806269758 -731863645
