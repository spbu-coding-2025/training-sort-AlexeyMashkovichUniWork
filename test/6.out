-1073523273 -1004689445 -983560054 -960328778 -896410198 -889082616 -867549762 -773010325 -771967523 -733348278 -731863645 -723686563 -692148342 -668749758 -626709420 -623216937 -522536860 -436421619 -433427300 -408512545 -339460166 -315695246 -303129302 -248768092 -245403077 -217981144 -136204352 -119720469 -3721327 51963406 58796479 79384943 125407446 147395584 224709397 249810896 266163929 280523379 283646186 310828051 408713394 422712389 456214497 469757932 469976483 511151302 533337182 559859324 567938229 586729240 591589240 647308108 658119998 796608042 805561389 806269758 883454023 935145623 944157591 960547330 975481150 1017927843 1018805020 1052208971
